docs/plugins/from-4-to-5.rst

```diff
@@ -0,0 +1,118 @@
+Plugin upgrade from 4 to 5 step by step
+=======================================
+
+Here is a list of steps that most of the Plugins may need to take to upgrade from Mautic 4 to Mautic 5. You should be able to get through each step, make a commit, move to the next one and once you are at finished you have upgraded your Plugin.
+
+1) Continuous Integration
```

Comment:
The aim was to have an ordered list of steps. Is that not possible to do in headings so each step has some details?

Reply:
We use the [Google Styleguide for Technical Writing](https://developers.google.com/style/headings#heading-and-title-format) as our styleguide, which specifically states:

> Don't include numbers in headings to indicate a sequence of sections.

Reply:
Hmm. Too bad. I like steps. I can then refer to them easily as "I got stuck on step 5". But I guess it doesn't make sense to disagree with Google...

Reply:
I expect that the preferred route would be to have a numerated list under a single heading, instead of using headings (which are used for navigation) to denote the numbered steps. It does look weird having the numbers in the navigation on the left when you build the docs, also.